### DISCUSSION TITLE
skoala/ms-gateway/service/manual-integrate/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1584] [Announcements]
# skoala/ms-gateway/service/manual-integrate/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/ms-gateway/service/manual-integrate/

**Comment by liyuerich** [comment]
手工接入 -添加服务  注册中心服务 现在 被以下2项代替：

接入注册服务中心 和注册配置中心服务

**Comment by Michelle951** [comment]
fixed by pull request: https://github.com/DaoCloud/DaoCloud-docs/pull/1892
